javascript/modules/client/test/integration/client-addressList.test.ts

```diff
@@ -475,8 +476,11 @@ describe("Client", () => {
       const pluginAddress = "0x1234567890123456789012345678901234567890";
       const action = client.encoding.addMembersAction(pluginAddress, members);
 
+      const functionFragment = getFunctionFragment(action.data)
+
       expect(typeof action).toBe("object");
       expect(action.data).toBeInstanceOf(Uint8Array);
+      expect(functionFragment.name).toBe("addWhitelistedUsers");
```

Comment:
Isn't it easier to simply do...?
```suggestion
      expect(bytesToHex(action.data)).toBe("0x.....__hardcoded_here__");
```

Reply:
The goal is preventing accidental changes. We don't need to test the decoding here